src/executor.cc

```diff
@@ -76,13 +77,14 @@ Result Executor::Execute(Engine* engine,
     }
   }
 
-  if (executionType == ExecutionType::kPipelineCreateOnly)
+  if (options->execution_type == ExecutionType::kPipelineCreateOnly)
     return {};
 
   // Process Commands
   for (const auto& cmd : script->GetCommands()) {
-    if (delegate && delegate->LogExecuteCalls())
-      delegate->Log(std::to_string(cmd->GetLine()) + ": " + cmd->ToString());
+    if (options->delegate && options->delegate->LogExecuteCalls())
```

Comment:
nit: this should have {}'s as the body is more then 1 line.

Reply:
Fixed